src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -430,4 +495,9 @@ const EmptyResultsBody = styled('div')`
   padding-bottom: ${space(2)};
 `;
 
+const StyledSelectControl = styled(SelectControl)`
+  width: 240px;
+  margin-right: ${space(2)};
```

Comment:
How about instead of adding margin and width to the `SelectControl` we use a grid instead of flex on the `FlexContainer` (rec renaming to `ActionContainer`)

```css
display: grid;
grid-template-columns: 240px max-content;
grid-gap: ${space(2)};